python/mdvtools/dbutils/mdv_server_app.py

```
@@ -2,74 +2,82 @@
 import time
 import json
 import shutil
+from datetime import datetime
```

Ruff (0.8.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)